docs/syntax/functions.md

```diff
@@ -122,19 +124,20 @@ type Maybe a
 
 2. **As an Extension Method:** A function defined _explicitly_ on an atom counts
    as an extension method on that atom. It can be defined on a typeset to apply
-   to all the atoms within that typeset.
+   to all the atoms within that typeset. An _extension_ method can be _static_
```

Comment:
`typeset`!? Looks like a relict of some old terminology. Also _"function defined _explicitly_ on an atom"_ doesn't seem to make any sense anymore. In both cases we should probably talk about **type**.

Reply:
Yes, that is a relic. Updated in 46df67bce940f3b56ced66ebd04ffd3b78239c3a